### DISCUSSION TITLE
How to get Audio At Faster Speeds than 4x?

### DISCUSSION THREAD

**Discussion by SarahSzabo** [smplayer-dev/smplayer#693] [Q&A]
Hello everyone, I watch video at accelerated speeds, and am interested in listening to lectures at > 4x, but the audio goes dead at faster than 4x acceleration.

Is there some setting somewhere I can change to get audio back at faster thaan 4x?

**Comment by gslin** [comment]
It's from mpv's default settings.  Change to mplayer will solve this.
